weis/aeroelasticse/openmdao_openfast.py

```diff
@@ -107,22 +108,167 @@ def setup(self):
-            self.add_input('r',                     val=np.zeros(n_span), units='m', desc='radial positions. r[0] should be the hub location \
+            # ElastoDyn and BeamDyn Inputs
+            self.add_input(
+                'r', 
+                val=np.zeros(n_span), 
+                units='m', 
+                desc='radial positions. r[0] should be the hub location \
                 while r[-1] should be the blade tip. Any number \
-                of locations can be specified between these in ascending order.')
-            self.add_input('le_location',           val=np.zeros(n_span), desc='Leading-edge positions from a reference blade axis (usually blade pitch axis). Locations are normalized by the local chord length. Positive in -x direction for airfoil-aligned coordinate system')
-            self.add_input('beam:rhoA',             val=np.zeros(n_span), units='kg/m', desc='mass per unit length')
-            self.add_input('beam:EIyy',             val=np.zeros(n_span), units='N*m**2', desc='flatwise stiffness (bending about y-direction of airfoil aligned coordinate system)')
-            self.add_input('beam:EIxx',             val=np.zeros(n_span), units='N*m**2', desc='edgewise stiffness (bending about :ref:`x-direction of airfoil aligned coordinate system <blade_airfoil_coord>`)')
-            self.add_input('x_tc',                  val=np.zeros(n_span), units='m',      desc='x-distance to the neutral axis (torsion center)')
-            self.add_input('y_tc',                  val=np.zeros(n_span), units='m',      desc='y-distance to the neutral axis (torsion center)')
-            self.add_input('flap_mode_shapes',      val=np.zeros((n_freq_blade,5)), desc='6-degree polynomial coefficients of mode shapes in the flap direction (x^2..x^6, no linear or constant term)')
-            self.add_input('edge_mode_shapes',      val=np.zeros((n_freq_blade,5)), desc='6-degree polynomial coefficients of mode shapes in the edge direction (x^2..x^6, no linear or constant term)')
-            self.add_input('gearbox_efficiency',    val=1.0,               desc='Gearbox efficiency')
-            self.add_input('gearbox_ratio',         val=1.0,               desc='Gearbox ratio')
-            self.add_input('platform_displacement', val=1.0,               desc='Volumetric platform displacement', units='m**3')
+                of locations can be specified between these in ascending order.',
+            )
+            self.add_input(
+                'le_location', 
+                val=np.zeros(n_span), 
+                desc='Leading-edge positions from a reference blade axis \
+                usually blade pitch axis). Locations are normalized by the \
+                local chord length. Positive in -x direction for airfoil-aligned coordinate system',
+            )
+            self.add_input(
```

Comment:
I think precomp computes the properties about the shear center? But when I checked how precomp computes the shear center, I'm not sure if it actually computes shear center. It looks more like computing elastic center to me.